docs/merged_reduced_scans_table.md

```diff
@@ -1,7 +1,14 @@
 # Reduced Table
 
-The table `firehook-censoredplanet.derived.merged_reduced_scans` contains filtered and pre-aggregated
-data to be used in the Censored Planet dashboard.
+The table `firehook-censoredplanet.derived.merged_reduced_scans_vN` (where N is a version number)
+contains filtered and pre-aggregated data to be used in the Censored Planet dashboard.
+
+This table is evolving and will change in backwards incompatible ways. When a backwards-incompatible
+change happens the version number will increment. Older version of the table will be deleted.
+
+The view `firehook-censoredplanet.derived.merged_reduced_scans` points to the latest version of
```

Comment:
Do you wanna name it `merged_reduced_scans_latest` to make it clearer what it is?
It may be confusing to tell the difference otherwise.

Reply:
I was hoping to to keep the old name so I don't have to go through and reconnect the data sources on all my side tables/old diagnostic queries.

Reply:
Ok, but you will still have to do that anyway, since the data change was a breaking change.